superset/assets/src/SqlLab/components/ExploreResultsButton.jsx

```diff
@@ -141,7 +141,9 @@ class ExploreResultsButton extends React.PureComponent {
         // open new window for data visualization
         exportChart(formData);
       })
-      .catch(() => {
+      .catch((e) => {
+        /* eslint no-console: 0 */
+        console.error(e);
```

Comment:
Did you intend to include these console messages?

Reply:
I did, but after reading a bit about best practices, I removed it...